product_security/red_hat_responsibilities/access_control.adoc

```diff
@@ -0,0 +1,105 @@
+[[security-access-control]]
+= Access Control
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-authentication]]
+== Authentication
+
+Authentication for the cluster is configured as part of the on-boarding process.
+{product-title} includes a built-in OAuth server. The OAuth server
+generates two kinds of tokens:
+
+* _Access tokens_ are longer-lived tokens that grant access to the API.
+* _Authorize codes_ are short-lived tokens; their only use is to be exchanged for
+an access token.
+
+You can integrate your cluster with various identity providers at installation
+time, or later by opening a support case.
+
+Supported identity providers include:
+
+* Keystone
+* Google Auth
+* GitHub OAuth
+* OpenID Connect
+* LDAP
+
```

Comment:
Add text and link: 
Learn more about Authentication in OpenShift Dedicated at : http://file.rdu.redhat.com/~ahardin/04232019/dedicated-security/architecture/additional_concepts/authentication.html